src/views/orb-view.ts

```diff
@@ -364,6 +390,80 @@ export class OrbView<N extends INodeBase, E extends IEdgeBase> implements IOrbVi
     }, 1);
   };
 
+  _handleKeyDown = (event: KeyboardEvent) => {
```

Comment:
This is missing on the `orb-map-view`.

Reply:
"orb-map-view" is already responding to key presses, so I decided not to make any changes.

Reply:
Awesome! Yep, that makes sense.